sdk/translation/Azure.AI.Translation.Document/src/Generated/DocumentTranslateContent.Serialization.cs

```diff
@@ -17,6 +17,13 @@ public partial class DocumentTranslateContent : IUtf8JsonSerializable, IJsonMode
     {
         void IUtf8JsonSerializable.Write(Utf8JsonWriter writer) => ((IJsonModel<DocumentTranslateContent>)this).Write(writer, ModelSerializationExtensions.WireOptions);
 
+        void IJsonModel<DocumentTranslateContent>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
+        {
+            writer.WriteStartObject();
+            JsonModelWriteCore(writer, options);
+            writer.WriteEndObject();
+        }
```

Comment:
As I said in [this comment](https://github.com/Azure/azure-sdk-for-net/pull/46895/files#r1825357024), if you follow the invocation starting here, you will find that the `start object` and `end object` are written twice, which is incorrect.

Reply:
Fixed